### DISCUSSION TITLE
Import error in create_client

### DISCUSSION THREAD

**Discussion by OmAvhad** [supabase/supabase#12798] [Questions]
I installed the supabase python library and initiliazed with the below code

```
import os
from supabase import create_client, Client

url: str = os.environ.get("SUPABASE_URL")
key: str = os.environ.get("SUPABASE_KEY")
supabase: Client = create_client(url, key)
```

but I am encountering this error

`ImportError: cannot import name 'create_client' from partially initialized module 'supabase' (most likely due to a circular import)`

**Comment by rick1290** [comment]
same

**Reply by xsa-dev** [reply]
same

**Comment by dvdpedros** [comment]
Hi! I had a similar issue and this other discussion helped 😄
https://github.com/supabase-community/supabase-py/issues/256

**Comment by RashmiRGowdaAOL** [comment]
This issue occurs when the code file name is also supabase. Try renaming you python file to some other name (ex. sup_base). It should work
